### ISSUE TITLE
Potential Breaking Change with New Release

### ISSUE THREAD

**Issue by RaffertyVIVE** [CodeSeven/toastr#553]
The file structure was changed for the files toastr.js and toastr.less they seem to have been moved out of the directory toastr/package/* to just toastr/toastr.*; which for us caused our builds to skip building css and triggered a package not found error.  

If anyone else has this issue we were able to fix this by removing the extra directory in the include statements.

**Comment by TimFerrell** [comment]
Thanks for reporting. I'll take a look at correcting the packaging and releasing a new rev.

**Comment by velichkov** [comment]
Yes, all files in `toastr/package/` were removed and now it is empty.
I'm not against such changes but please increase the version accordingly.
